docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst

```diff
@@ -19,37 +19,27 @@ or more loose stools in a 24-hour period." (p. 88 of
 Modeling Diarrheal Diseases in GBD 2017
 ---------------------------------------
 
+The GBD diarrheal diseases model follows a standard GBD framework, including a cause of death (CoD) model and
+a nonfatal model.
 
+The CoD model estimates the cause-specific mortality rate (CSMR) within the total population, and a cause fraction.
+These estimates are based on vital registration and autopsy data.
 
-
-According to the [GBD-2017-YLD-Capstone-Appendix-1]_, "There are no
-major modelling updates from GBD 2016," (p. 93) and "self-reported
-prevalence is the reference category" (p. 88).
-
-.. todo::
-
-   Add more context regarding GBD 2016 model for people who are not familiar.
-
-Regarding the duration of a bout of diarrhea, "the mean duration was
-the duration in days, an average of 4.3 (4.2 4.4)". (p. 89, based on a
-paper referenced there).
-For GBD 2017, the remission period was modeled as 5 days.
-Since this assumption gets into the DisMod
-model, we will use the remission rate that comes from DisMod.
+The nonfatal model is run in DisMod. The primary inputs are prevalence epi data, for which self-reported prevalence is
+the reference definition, and the CSMR estimates from CodCorrect. Separately,
+the ratio of mild/moderate/severe diarrhea is estimated, based on data from a
+systematic review. These estimates do not vary by age/sex/location/year, and are applied to the prevalence
```

Comment:
These _severity ratio estimates_ do not vary by ... --- to make it clear that you are not also talking about the DisMod or CodCorrect models.

Reply:
It's a surpise to me that they are constant across demography.  Did you confirm this in the data? I thought they were based on models of healthcare utilization in the country.  @ctroeger your expertise is definitely appreciated here.

Reply:
@ctroeger said that the parent model has a flat estimate for severities. those flat estimates _are_ based on a systematic review which relies in part on hospitalization. The etiologies are far more complicated--and that was where, for their CoD model, they assumed that severe and fatal cases=hospitalized cases @collijk